deploy_packages.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/python
+
+import sys
+import os
+import shutil
+from shutil import copyfile
+
+deployable_packages = ['common', 'datamodel', 'parsers', 'gl_extensions', 'input_manager', 'renderer', 'ticker', 'traits', 'window_manager', 'camera'] # left for later use
```

Comment:
This should probably be read from a conf file anyway, so I see little reason to leave it be